app/assets/javascripts/components/story/CollapsedStory/CollapsedStoryFocusButton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const disableToggle = event => {
+  event.stopPropagation();
+}
+
+const CollapsedStoryFocusButton = ({ onClick }) =>
+  <span onClick={disableToggle}>
```

Comment:
Why was this function necessary? Is it so the click doesn't propagate to the story and make it expanded? It looks a little tricky to me, kinda hard to understand that at first sight. What do you think about this:

```js
const CollapsedStoryFocusButton = ({ onClick }) => {
  const handeClick = (e) => {
    e.stopPropagation();
    return onClick(e);
  };

  return (
    <button type="button" className="btn btn-default locate-btn" onClick={onClick}>
      <i className="mi md-gps-fixed md-14">gps_fixed</i>
    </button>
  )
}